.multi-thread.yml

```diff
@@ -0,0 +1,33 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+
+pr:
```

Comment:
Let's not have this for PRs for now please, there isn't enough CI capacity.

Reply:
Good point!